integration-tests/jest-puppeteer.config.js

```diff
@@ -13,6 +13,7 @@ const printWarning = message => {
 }
 
 module.exports = {
+  exitOnPageError: false,
```

Comment:
`exitOnPageError`: Do not exits page on any JS page error message thrown by UI code. Defaults is true.

Reply:
Why do we want to make this false? I assume it is true by default to maximize the chance of catching errors.

Reply:
`exitOnPageError: true` makes tests flaky: It's not a good practice to allow sudden/random page exits during running of tests. 

It would be good to log page errors (in future) so it would be easier to know what happened while letting tests finish from beginning to end.

Reply:
With the true option, is the error that caused the page exit present in the test artifacts?

Reply:
no, I don't think page errors are saved anywhere. Ideally, page console errors and networks requests and responses should be logged to aid with the failure troubleshooting.

Reply:
> `exitOnPageError: true` makes tests flaky: It's not a good practice to allow sudden/random page exits during running of tests.

I'm not sure I follow this logic. I haven't personally experimented with this, but I assume this means that, if an exception bubbles up to the top, that would cause the test to exit. However, we generally catch these kinds of errors and display them as the error alerts that appear in the upper right. This generally results in the test failing. If we happen to _not_ catch an error and display a notification, then the page "crashes" and the test will fail. I can't think of a case where exiting after an exception would cause a test to fail while _not_ exiting after an exception might lead to the test passing.